### PR TITLE
 Update test suite determination logic

### PR DESCRIPTION
**Pull request by tbiethman**
### Summary
The existing logic has a couple holes that cause builds to fail (e.g. having 9 packages causes suite4 to always be empty). The new expression should always ensure every suite has at least one test set. 

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -26,4 +27,5 @@ Cerner Corporation
 [@alex-bezek]: https://github.com/alex-bezek
 [@noahbenham]: https://github.com/noahbenham
 [@BenBoersma]: https://github.com/BenBoersma
-[@dvv297]: https://github.com/dv297
+[@dv297]: https://github.com/dv297
```
Comment (tbiethman) [inline CONTRIBUTORS.md:30]: Fixed Daniel's link too

**Review by emilyrohrbough — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

**Review by yuderekyu — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
@tbiethman looks like there are a few lint errors

**Comment by emilyrohrbough** [comment]
Also, you should add yourself to the Contributors for this repo 😄

**Comment by tbiethman** [comment]
Boo. Note to linter: order of operations is a thing.

**Pull request merged**